erpnext/selling/doctype/sales_order_item/sales_order_item.py

```diff
@@ -12,4 +12,14 @@ def __setup__(self):
 		print_settings_for_item_table(self)
 
 def on_doctype_update():
-	frappe.db.add_index("Sales Order Item", ["item_code", "warehouse"])
\ No newline at end of file
+	frappe.db.add_index("Sales Order Item", ["item_code", "warehouse"])
+
+@frappe.whitelist()
```

Comment:
Please add a docstring here, especially since this is an API endpoint. Refer to the auto doctring generator

https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring